libs/coin-evm/src/api/explorer/etherscan.ts

```diff
@@ -69,6 +70,9 @@ export const getLastCoinOperations = async (
   if (fromBlock) {
     url += `&startBlock=${fromBlock}`;
   }
+  if (toBlock) {
```

Comment:
More of a personal pref but i'd personally compare with undefined or null value, if(0) doesn't execute instruction, same for ''

Reply:
In this case I also want to "ignore" 0 actually, because it could break the request completely depending on the implementation:

"get block from 100 to 0" ->  doesn't make sense, we should ignore the "to 0"
"get block from 0 to 100" -> uncessary to add as default behaviour is already "null | undefined = 0" for the "startBlock" option
"get block from 0 to 0" -> this one is tricker because you would expect nothing as a response but will get everything from block 0 to latest instead, but I guess this is really an edge case that don't need to be considered right now as nothing in the coin integration would use it this way without everything being broken before this request :p